internal/measurexlite/tls.go

```diff
@@ -70,8 +79,17 @@ func (tx *Trace) OnTLSHandshakeDone(started time.Time, remoteAddr string, config
 	}
 
 	select {
-	case tx.networkEvent <- NewAnnotationArchivalNetworkEvent(
-		tx.Index(), t, "tls_handshake_done", tx.tags...):
+	case tx.networkEvent <- NewArchivalNetworkEvent(
+		tx.Index(),
+		t,
+		"tls_handshake_done",
+		"tcp",
+		remoteAddr,
+		0,
+		nil,
+		t,
+		tx.tags...,
+	):
```

Comment:
This is such that we have the endpoints in this event (because... why not? It helps with sorting and I don't see why omitting a piece of information we know would ever be wrong).